Makefile

```diff
@@ -24,19 +24,37 @@ build/llvm.BUILT:
 		-DLLVM_DEFAULT_TARGET_TRIPLE=wasm32-wasi \
 		-DLLVM_EXTERNAL_CLANG_SOURCE_DIR=$(LLVM_PROJ_DIR)/clang \
 		-DLLVM_EXTERNAL_LLD_SOURCE_DIR=$(LLVM_PROJ_DIR)/lld \
-		-DLLVM_ENABLE_PROJECTS="lld;clang" \
+		-DLLVM_EXTERNAL_CLANG_TOOLS_EXTRA_SOURCE_DIR=$(LLVM_PROJ_DIR)/clang-tools-extra \
+		-DLLVM_ENABLE_PROJECTS="lld;clang;clang-tools-extra" \
 		-DDEFAULT_SYSROOT=$(PREFIX)/share/wasi-sysroot \
+		-DLLVM_INSTALL_BINUTILS_SYMLINKS=TRUE \
 		$(LLVM_PROJ_DIR)/llvm
 	ninja -v -C build/llvm \
 		install-clang \
+		install-libclang \
+		install-clang-format \
+		install-clang-tidy \
+		install-clangQuery \
+		install-clang-apply-replacements \
```

Comment:
Do all of these targets existing llvm 8 too?   Do want want to continue to support llvm-8?

Reply:
I haven't tested a full build, but they're all resent in the llvm 8 source tree.